src/lib/list/_list-theme.scss

```diff
@@ -8,14 +8,22 @@
   $background: map-get($theme, background);
   $foreground: map-get($theme, foreground);
 
-  .mat-list, .mat-nav-list {
+  .mat-list, .mat-nav-list, .mat-selection-list {
     .mat-list-item {
       color: mat-color($foreground, text);
     }
 
+    .mat-list-option {
+      color: mat-color($foreground, text);
+    }
+
     .mat-subheader {
       color: mat-color($foreground, secondary-text);
     }
+
+    .mat-list-item-disabled {
+      background: #ededed;
```

Comment:
:+1: changed to `background-color: mat-color($background, disabled-list-option); `